### ISSUE TITLE
A way to hard set a number for the terrain height limit

### ISSUE THREAD

**Issue by TheSchmeat** [PG85/OpenTerrainGenerator#508]
https://gyazo.com/9f7e9d64e0d42bbfe3f28a4158080d51
![caa543955cc06c161a987b0189f27f1d](https://user-images.githubusercontent.com/69754567/90333738-2958df00-df96-11ea-94b9-b164d376a800.png)

I'm trying to set the number somewhere around 200 rather than the hard-set 128 or 256 depending on if you select 7 or 8 as the WorldHeightCapBits. I'd like to be able to set a specific height rather than putting a number into the equation. If this is not possible, is there anywhere in the files I can go to edit the formula that is used to calculate that limit? My problem is I want to set it to 8 because I want my mountains to be extremely tall and diverse but they often times end up reaching the height limit of 256 so I am unable to build on the tops of them. I'd like to find a happy medium where I can limit the heights to around something like 200. 128 is too short for the tallest mountains but 256 is too tall to build on. Thank you for your time in advance. Picture for reference of formula I am asking about. Thank you very much for your time!

TL;DR: Need to find a way to set terrain height cap manually so I can cap my mountains somewhere between the two current options of 128 and 256.

**Comment by BiomeBundle** [comment]
This isnt possible I don't think. It's not really needed, I've never had a problem limiting a biome to whatever height by tweaking biome config settings.

**Comment by TheSchmeat** [comment]
Thanks for the quick response! Could you give an example of how you might set your config in order to limit the mountain tops to around y=200? I assume you'd use the CustomHeightControl options, however, I'm not entirely sure how the CustomHeightControl works as the description is kind of confusing so I'd greatly appreciate if you could briefly explain it a bit so I may understand. Also, if you have a moment to explain what exactly Volatility 1 and 2 are? Is it basically Horizontal(1) and Vertical(2) fracturing settings for specific biomes?

Here is a picture of my current configuration for Extreme Hills+ M.
![64b5793ab6b118b1368f41e7693a4ad5](https://user-images.githubusercontent.com/69754567/90344198-5bddf880-dfe5-11ea-89d6-cf981d2f554e.png)

**Comment by BiomeBundle** [comment]
Probably better to discuss this on the discord https://discord.gg/UXzdVTH

**Comment by TheSchmeat** [comment]
Thanks! I just joined. May I reach out to your through the discord?

**Comment by PG85** [comment]
Hey @TheSchmeat , looks like a settings thing, just ask around on the OTG discord, the community or Team OTG members should be able to help. Thanks!
